x-pack/plugins/reporting/types.d.ts

```diff
@@ -107,11 +107,31 @@ export interface CryptoFactory {
   decrypt: (headers?: Record<string, string>) => string;
 }
 
+export interface TimeRangeParams {
+  timezone: string;
+  min: Date | string | number;
+  max: Date | string | number;
+}
+
+type PostPayloadState = Partial<{
+  state: {
+    query: any;
+    sort: any[];
+    columns: string[]; // TODO
```

Comment:
- `columns` is not passed by the client currently.
- Related: https://github.com/elastic/kibana/pull/34571/files#r275093173